service/src/main/java/uk/gov/hmcts/reform/fpl/enums/DocmosisTemplates.java

```diff
@@ -3,7 +3,8 @@
 public enum DocmosisTemplates {
     C6("FL-PLW-HNO-ENG-00092.docx", "Notice_of_proceedings_c6.pdf"),
     C6A("FL-PLW-HNO-ENG-00093.docx", "Notice_of_proceedings_c6a.pdf"),
-    SDO("FL-PLW-STD-ENG-00099.doc", "standard_directions_order.pdf");
+    SDO("FL-PLW-STD-ENG-00099.doc", "standard_directions_order.pdf"),
+    C21("FL-PLW-GOR-ENG-00218.docx","C21_Order_");
```

Comment:
space after the comma...

Reply:
to be fair the intelliJ thing displaying "documentTitle" made this one a bit hidden 😄